selene/schema/results/fields.py

```diff
@@ -58,14 +70,61 @@ def resolve_details(root, _info):
         return details.findall('detail')
 
 
+class ResultNVT(ScanConfigNVT):
+    class Meta:
+        # default resolver is not inherited. Must be declared
+        default_resolver = text_resolver
+
+    version = graphene.String(description='Version of the NVT used in the scan')
+
+    def resolve_version(root, _info):
+        return get_text_from_element(root, 'version')
+
+
+class ResultCVE(graphene.ObjectType):
+    oid = graphene.String(name='id')
+    severity = graphene.Field(SeverityType)
+    score = graphene.Int()
```

Comment:
Can be removed. We discussed the score int value and came to the conclusion that we are not going to implement it completely and instead remove it from gvmd and gsa again.
```suggestion
```